package.json

```diff
@@ -28,7 +30,8 @@
     "mocha": "^6.1.4",
     "nyc": "^14.1.1",
     "prettier": "^1.15.2",
-    "should": "^8.0.2"
+    "should": "^13.2.3",
```

Comment:
I have upgraded should.js to latest version where typescript definitions are presented